src/app.js

```diff
@@ -127,13 +127,13 @@ class Messenger extends EventEmitter {
           // The page does not have a public profile and calling the Graph API here will always yield a 400.
           session.profile = {};
           return session;
-        } else {
-          return this.getPublicProfile(messagingEvent.sender.id)
-            .then((profile) => {
-              session.profile = profile;
-              return session;
-            });
         }
+
+        return this.getPublicProfile(messagingEvent.sender.id)
```

Comment:
yeah, this came up as a lint warning with the updated rules, and cleaned it up as a boy scout rule http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule